### ISSUE TITLE
Feature request: show full title on hover

### ISSUE THREAD

**Issue by protist** [kotelnik/plasma-applet-active-window-control#23]
Hi, I don't have a lot of room for this plasmoid in my panel. Generally that is fine; I can get the gist of the window title from the plasmoid.

![screenshot_at_20160914_16 32 49](https://cloud.githubusercontent.com/assets/646668/18502203/161fa8c6-7a99-11e6-8aa9-5de3a34beddb.png)

However, in some cases, it's useful to know the full title of the window. For these rare cases, perhaps it would be possible to show the full title of the window when I hover over the plasmoid?

**Comment by kotelnik** [comment]
It's a good idea, I'll work on it, thanks!

**Comment by kupiqu** [comment]
I actually like this too, and it could be combined with issue #21 so, if enabled as an option, just the app appears in the applet and full title on hovering... does it sound appropriate?

**Comment by protist** [comment]
I personally would only ever want the window title displayed; I find the application icon sufficient to work out what application is running. However, as long as a fix to #21 is just an option, I'm not adverse to it.

**Comment by kupiqu** [comment]
I don't actually use the icons, neither in the windows neither in the applet, I prefer the name, that is why I ask for that option :)

**Comment by kotelnik** [comment]
Implemented (tooltip or text shrink) in latest commits. Please reopen when necessary.

**Comment by protist** [comment]
Works great! Thank you for the quick and effective fix. I also love the automatic text shrink option. It's a very elegant solution IMO.

**Comment by kotelnik** [comment]
Thanks, I'm glad You like it. :)

**Comment by protist** [comment]
Definitely! However, I have one minor quibble with the text shrink. Would it be possible for users to specify a minimum font size? In some cases, the text is way too small to be useful.

![img tiny text](http://i.imgur.com/rzC8IHQ.png)

It looks like you've restricted it to two lines, which seems reasonable for a horizontal panel, at least. However, once the font size gets beyond `threshold` points, perhaps an ellipsis could be shown, as per the elide option. (Also, thank you for introducing me to a new word. :smiley:  )

**Comment by kotelnik** [comment]
You're right, it needs to be tuned, thanks for feedback!
